release/train_tests/benchmark/image_classification/localfs_image_classification_jpeg/factory.py

```diff
@@ -60,6 +61,7 @@ def get_ray_datasets(self) -> Dict[str, ray.data.Dataset]:
                 base_dir=LOCALFS_JPEG_SPLIT_DIRS["val"],
                 field_names=["class"],
             ),
+            parallelism=20,
```

Comment:
* let's parameterize this config
* let's use `override_num_blocks`. `parallelism` is deprecated.